ddl/restart_test.go

```diff
@@ -104,6 +113,31 @@ LOOP:
 	}
 }
 
+func testRunInterruptedJobT(t *testing.T, d *ddl, job *model.Job) {
+	done := make(chan error, 1)
+	go runInterruptedJob(d, job, done)
+
+	ticker := time.NewTicker(d.lease * 1)
+	defer ticker.Stop()
+LOOP:
+	for {
+		select {
+		case <-ticker.C:
+			err := d.Stop()
+			require.NoError(t, err)
+			d.restartWorkers(context.Background())
+			time.Sleep(time.Millisecond * 20)
+		case err := <-done:
+			require.NoError(t, err)
+			break LOOP
+		}
+	}
+}
+
+func testCheckOwnerT(t *testing.T, d *ddl, expectedVal bool) {
+	require.Equal(t, expectedVal, d.isOwner())
+}
+
 func (s *testSchemaSuite) TestSchemaResume(c *C) {
```

Comment:
Can you help this function to support testify?

Reply:
Ditto ```TestSchemaResume```

Reply:
OK, done

PTAL

Reply:
NOTE that `getDDLSchemaVer` is deleted 'cause linter will warning that the method is not used.